test/runner/lib/cli.py

```diff
@@ -28,6 +28,7 @@
 
 from lib.executor import (
     command_posix_integration,
+    command_aws_ssm_integration,
```

Comment:
Revert this change, and all other changes to `test/runner/lib/`.

Reply:
Done.